Makefile

```diff
@@ -56,6 +56,13 @@ ut: calicobuild.created
 	'/tmp/etcd -data-dir=/tmp/default.etcd/ >/dev/null 2>&1 & \
 	nosetests calico_containers/tests/unit -c nose.cfg'
 
+ut_kubernetes: calicobuild.created
```

Comment:
Could you add this to the "test" clause above so that it gets run when the other tests are run?

Also, just for consistency, this should probably be called `ut-kubernetes` instead of `ut_kubernetes`

e.g
```
test: ut st ut-kubernetes
```